app/model/NotificationTestType.scala

```diff
@@ -17,11 +17,20 @@
 package model
 
 import model.ApplicationStatus.ApplicationStatus
-import model.ApplicationStatus.{ PHASE1_TESTS_FAILED, PHASE2_TESTS_FAILED, PHASE3_TESTS_FAILED}
+import model.ApplicationStatus.{ PHASE1_TESTS_FAILED, PHASE2_TESTS_FAILED, PHASE3_TESTS_FAILED, READY_FOR_EXPORT }
 import model.ProgressStatuses._
 
+sealed trait NotificationTestType {
```

Comment:
The idea is that you can have different types of notification (failed, success, expired). Let me know if it make sense otherwise I'll change it